scripts/perlin_noise.lua

```diff
@@ -30,6 +31,7 @@ perlin.gy = {}
 perlin.randMax = 256
 perlin.loaded = false
 
+--- The method load.
```

Comment:
meaningless redundant comment is redundantly meaningless.

Documentation should state "why" not "what" the "what" is already in the code.